starter-code/scripts/articleView.js

```diff
@@ -60,6 +75,16 @@ articleView.setTeasers = function() {
 
     // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
   */
+  $('.read-on').on('click', function(event) {
+    event.preventDefault();
+    $(this).prev().find('*').show();
```

Comment:
Me too! Thanks for your help. Trial and error process is good for learning some of the quirks :)